docs/source/api/queries.rst

```diff
@@ -222,14 +230,20 @@ Request Structure
     :widths: 15, 30, 20, 15
 
     "Account ID", "account id to request transactions from", "<account_name>@<domain_id>", "makoto@soramitsu"
+    "Page size", "size of the page to be returned by the query, if response contains less transactions than page size, then next tx hash will be empty in response", "page_size > 0", "5"
+    "First tx hash", "hash of the first transaction in the page. If that field is not set, then first transactions are returned", "hash in hex format", "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"
```

Comment:
Maybe write that if no hash is specified, return transactions **starting from the first**